client-js/app/elements/labrad-grapher.ts

```diff
@@ -35,30 +52,46 @@ export class LabradGrapher extends polymer.Base {
     this.showStars = false;
     this.showTrash = false;
     this.updateButtons();
+
+    this.initializeListItems_(
+      this.path,
+      this.dirs,
+      this.datasets);
+    this.applyListFilter_();
   }
 
   starClicked() {
     this.showStars = !this.showStars;
     if (this.showStars) this.showTrash = false;
     this.updateButtons();
-    this.kick++;
+    this.applyListFilter_();
```

Comment:
Google style guide for JavaScript (and many languages) has all private members and private methods suffixed with an underscore. I can't seem to find exactly where it is called out, but it is common across the style guides.

https://google.github.io/styleguide/javascriptguide.xml?showone=Visibility__private_and_protected_fields_#Visibility__private_and_protected_fields_
https://google.github.io/styleguide/cppguide.html

I have tried to make new code at least follow the style guide, which leads to a weird mash of styles, but it's somewhat unavoidable without every PR first requiring a PR where the entire file gets refactored which are burdensome to review. If this is the workflow we'd prefer moving forward, we can do that.